examples/dask/special-topics-rolling-average.ipynb

```diff
@@ -0,0 +1,179 @@
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-01-25T17:33:22.602911Z",
+     "iopub.status.busy": "2022-01-25T17:33:22.602324Z",
+     "iopub.status.idle": "2022-01-25T17:33:25.666206Z",
+     "shell.execute_reply": "2022-01-25T17:33:25.665688Z",
+     "shell.execute_reply.started": "2022-01-25T17:33:22.602827Z"
+    },
+    "tags": []
+   },
+   "source": [
+    "> \"I need to calculate a rolling average of a numerical column, in time series data. In pandas, I can do this with rolling(x).mean() with sorted values, but what do I do in Dask, with distributed data?\"\n",
+    "\n",
+    "This example will walk you through performing rolling average calculations with data that is distributed over a Dask cluster. We'll use New York City taxi trip data, and get the 30-day rolling average of base fare prices, for our example.\n",
+    "\n",
+    "First, start the Dask cluster associated with your Saturn Cloud resource."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "991b4299-f93d-441f-9a42-46d4aca332b0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask_saturn import SaturnCluster\n",
+    "from dask.distributed import Client\n",
+    "\n",
+    "client = Client(SaturnCluster())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a0e7f9e-8c67-4cd0-adf4-1d082addb1c5",
+   "metadata": {},
+   "source": [
+    "After running the above command, it's recommend that you check on the Saturn Cloud resource page that the Dask cluster as fully online before continuing. Alternatively, you can use the command `client.wait_for_workers(3)` to halt the notebook execution until all three of the workers are ready.\n",
+    "\n",
+    "Now load your data into a Dask Dataframe. Here we are loading data for csv file located at s3 storage. Using read_csv from Dask takes the same form as using that function from pandas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7df4bb88-37cc-4b8a-bb9d-b476102b133a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import dask.dataframe as dd\n",
+    "\n",
+    "taxi = dd.read_csv(\n",
+    "    \"s3://nyc-tlc/trip data/yellow_tripdata_2019-01.csv\",\n",
+    "    parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"],\n",
+    "    storage_options={\"anon\": True},\n",
+    ").sample(frac=0.1, replace=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a2886561-f4be-481d-95c1-82997a609ae9",
+   "metadata": {},
+   "source": [
+    "For dataframe we are calling function `set_index` to set `tpep_pickup_datetime` columns as the index. This will sort data by index within and across partitions. At this point, data and labels are lazy collections. They won’t be read into the workers' memory until some other computation asks for them. Hence, before we pass this data to other tasks we will call `presist()`. This will ensure that loading of data is run only once."
```

Comment:
```suggestion
    "For the DataFrame we are calling function `set_index` to set `tpep_pickup_datetime` columns as the index. This will sort data by index within and across partitions. At this point, data and labels are lazy collections. They won’t be read into the workers' memory until some other computation asks for them. Hence, before we pass this data to other tasks we will call `presist()`. This will ensure that loading of data is run only once."
```